schemas/PreSourceIlluminationCorrection.yaml

```diff
@@ -0,0 +1,189 @@
+# This file defines the mapping between the columns in a single visit+detector
+# source table and their respective DPDD-style column names, as used by
+# `lsst.pipe.tasks.postprocess.TransformSourceTableTask`.
+# See the DPDD for more information about the output: https://lse-163.lsst.io
+funcs:
+    sourceId:
+        functor: Column
+        args: id
+    coord_ra:
+        # reference position required by db. Not in DPDD
+        functor: CoordColumn
+        args: coord_ra
+    coord_dec:
+        # Reference position required by db. Not in DPDD
+        functor: CoordColumn
+        args: coord_dec
+    parentSourceId:
+        functor: Column
+        args: parent
+    x:
+        functor: Column
+        args: slot_Centroid_x
+    y:
+        functor: Column
+        args: slot_Centroid_y
+    xErr:
+        functor: Column
+        args: slot_Centroid_xErr
+    yErr:
+        functor: Column
+        args: slot_Centroid_yErr
+    ra:
+        functor: RAColumn
+    dec:
+        functor: DecColumn
+    raErr:
+        functor: RAErrColumn
+    decErr:
+        functor: DecErrColumn
+    ra_dec_Cov:
+        functor: RADecCovColumn
+    psfFlux:
+        functor: Column
+        args:
+            - slot_PsfFlux_flux
```

Comment:
The others are all scalar entries, but these two are single element lists?  I see Gaussian fluxes are the same below, so maybe this is just how flux/flux errors work?

Reply:
Yeah.  This is just copied from the official functor file, slimmed down for our needs.